indico/modules/events/papers/controllers/management.py

```diff
@@ -123,6 +125,23 @@ def _process(self):
                                               form=form)
 
 
+class RHManageContact(RHManagePapersBase):
+    """Allow to send emails to members of reviewing team"""
+    def _process(self):
+        cfp = self.event_new.cfp
+        paper_persons_dict = defaultdict(lambda: {'judge': False, 'content_reviewer': False, 'layout_reviewer': False})
+        for judge in cfp.judges:
```

Comment:
Do you know how to reduce the three loops to one? I'm quite sure that this could be done in less lines of code but don't know how.